### PR TITLE
updated Create Course Page and tests

### PR DESCRIPTION
**Pull request by staubina**
Updates to the Create Course Page to new format to better support snapshot tests.

### CODE REVIEW

File: src/components/CreateCoursePage/CreateCourseForm.jsx [hunk 1]
```diff
@@ -169,7 +169,8 @@ CreateCourseForm.propTypes = {
   isCreating: PropTypes.bool,
 };
 
-export default reduxForm({
+const CreateCourseForm = reduxForm({
```
Comment (bbaker6225) [inline src/components/CreateCoursePage/CreateCourseForm.jsx]: I would do `export default CreateCourseForm` and then do a regular export on the base, that way you only have to update imports on the test file

File: src/components/CreateCoursePage/CreateCourseForm.test.jsx [hunk 2]
```diff
@@ -1,18 +1,28 @@
...
+const organizations = [
+  { name: 'edX', key: 'edx' }, { name: 'edX2', key: 'edx2' },
+];
 
 describe('CreateCourseForm', () => {
   it('renders html correctly with no orgs', () => {
-    const component = shallow(<CreateCourseForm />);
+    const component = shallow(<BaseCreateCourseForm
+      handleSubmit={() => {}}
+      pristine
```
Comment (bbaker6225) [inline src/components/CreateCoursePage/CreateCourseForm.test.jsx]: pristine and submitting seem like they should be default props like I did here, but personal opinion https://github.com/edx/publisher-frontend/blob/5c4253d9f6e2450f0de3425b5015ef0067055d4e/src/components/EditCoursePage/EditCourseForm.jsx#L268-#L272

File: src/components/CreateCoursePage/CreateCourseForm.jsx (hunk 1 above)
Comment (staubina) [reply src/components/CreateCoursePage/CreateCourseForm.jsx]: 👍 Was following the other examples.  I can adjust this.

File: src/components/CreateCoursePage/CreateCourseForm.test.jsx (hunk 2 above)
Comment (staubina) [reply src/components/CreateCoursePage/CreateCourseForm.test.jsx]: I will take a look.

File: src/components/CreateCoursePage/CreateCourseForm.jsx (hunk 1 above)
Comment (bbaker6225) [reply src/components/CreateCoursePage/CreateCourseForm.jsx]: Yeah, that's how @inventhouse and I did it on the `CreateInstructorForm`, but we should have done export default on the connected one

Comment (bbaker6225) [reply src/components/CreateCoursePage/CreateCourseForm.jsx]: Sorry for the confusion

Comment (staubina) [reply src/components/CreateCoursePage/CreateCourseForm.jsx]: No worries.

File: src/components/CreateCoursePage/CreateCourseForm.jsx [hunk 3]
```diff
@@ -164,13 +166,14 @@ BaseCreateCourseForm.propTypes = {
...
+  pristine: PropTypes.bool,
+  submitting: PropTypes.bool,
   isCreating: PropTypes.bool,
 };
 
 const CreateCourseForm = reduxForm({
   form: 'create-course-form',
 })(BaseCreateCourseForm);
-export { BaseCreateCourseForm, CreateCourseForm };
+export default CreateCourseForm;
```
Comment (bbaker6225) [inline src/components/CreateCoursePage/CreateCourseForm.jsx]: no need to change just an fyi, you can just do export default CreateCourseForm = reduxForm...

Comment (staubina) [reply src/components/CreateCoursePage/CreateCourseForm.jsx]: Ok, I thought it would be more clear to have both exports below.  I will adjust.

Comment (staubina) [reply src/components/CreateCoursePage/CreateCourseForm.jsx]: Need to rebase anyway

**Pull request merged**